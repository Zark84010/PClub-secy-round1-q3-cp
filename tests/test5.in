1 537 20
431172 
c 1 1616
f 1 1
f 1 1
c 1 2020
f 1 1
f 1 1
c 1 909
f 1 1
f 1 1
c 1 1616
f 1 1
f 1 1
c 1 1717
f 1 1
f 1 1
c 1 1818
f 1 1
f 1 1
c 1 2121
f 1 1
