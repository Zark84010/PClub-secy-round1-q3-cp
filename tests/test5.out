867792
867792
1084740
1084740
487224
487224
867792
867792
922029
922029
972630
972630
1136856
